File: week-04/day-03/Testing/src/Fibonacci/FibonacciTest.java
package Fibonacci;

import org.junit.Test;

import static org.junit.Assert.assertEquals;

public class FibonacciTest {
  @Test
  public void TestCorrectNumber() {
    assertEquals(8, Fibonacci.fibonacci(6));
  }

  @Test
  public void TestForZero() {
    assertEquals(0, Fibonacci.fibonacci(0));
  }

  @Test
  public void TestForNegative() {
    assertEquals(0, Fibonacci.fibonacci(-20));
  }
}

File: CSharp/EntityFramework/EntityTest/FibonacciTest.cs
using EntityFramework;
using NUnit.Framework;

namespace EntityTest
{
    [TestFixture]
    public class FibonacciTest
    {
        [Test]
        public void TestCorrectNumber()
        {
            Assert.AreEqual(8, Program.fibonacci(6));
        }
        
        [Test]
        public void TestForZero()
        {
            Assert.AreEqual(0, Program.fibonacci(0));
        }

        [Test]
        public void TestForNegative()
        {
            Assert.AreEqual(0, Program.fibonacci(-20));
        }

    }
}